cmd/sidecars/README.md

```diff
@@ -21,7 +21,9 @@ of the container. This binary will search in $PATH for binaries named after the
 onDefineDomain) and run them and provide the necessary arguments as command line options (flags).
 
 Besides a binary, one could also execute shell or python scripts by making them available at the
-expected location.
+expected location. For this use case, a generic `sidecar-shim-image` image that is able to execute
+Bash and Python scripts is built as part of the build chain and `virt-operator`, and so `virt-controller`,
+are configured with a reference to it via an environmental variable.
```

Comment:
This section is 100% on `sidecar-shim` image so I'd just put in another paragraph that this is default image, provided by kubevirt (since version 1.3.0)

Reply:
done